_playwright-tests/tests/API/Repositories.spec.ts

```diff
@@ -530,4 +530,69 @@ test.describe('Repositories', () => {
       expect(versionFilterUuids).not.toContain(repo3.uuid);
     });
   });
+
+  test('Partial update repository', async ({ client, cleanup }) => {
+    const repoName = `patch-repository-${randomName()}`;
+    const repoUrl = randomUrl();
+    const updatedName = `updated-${repoName}`;
+    const updatedUrl = 'https://content-services.github.io/fixtures/yum/comps-modules/v1/';
+
+    await cleanup.runAndAdd(() => cleanupRepositories(client, repoName, updatedName));
+
+    let repo: ApiRepositoryResponse;
+    await test.step('Create a repository for updating', async () => {
+      repo = await new RepositoriesApi(client).createRepository({
+        apiRepositoryRequest: {
+          name: repoName,
+          url: repoUrl,
+          distributionArch: 'x86_64',
+          distributionVersions: ['8'],
+          metadataVerification: true,
+        },
+      });
+      expect(repo.name).toBe(repoName);
+      expect(repo.url).toBe(repoUrl);
+      expect(repo.distributionArch).toBe('x86_64');
+      expect(repo.distributionVersions).toEqual(['8']);
+      expect(repo.metadataVerification).toBe(true);
+    });
+
+    await test.step('Update repository name and URL', async () => {
+      const updatedRepo = await new RepositoriesApi(client).partialUpdateRepository({
+        uuid: repo.uuid!,
+        apiRepositoryUpdateRequest: {
+          name: updatedName,
+          url: updatedUrl,
+        },
+      });
+
+      expect(updatedRepo.uuid).toBe(repo.uuid);
+      expect(updatedRepo.name).toBe(updatedName);
+      expect(updatedRepo.url).toBe(updatedUrl);
+      expect(updatedRepo.distributionArch).toBe('x86_64');
```

Comment:
i would add assertions for the module_hotfixes and gpg_key here too just since you're updating them in the following test case

Reply:
they're not set during creation, so they'd just be the empty values, but yeah might as well